protos/feast/core/InfraObjects.proto

```diff
@@ -0,0 +1,36 @@
+//
+// * Copyright 2021 The Feast Authors
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     https://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+//
+
+syntax = "proto3";
+
+package feast.core;
+option java_package = "feast.proto.core";
+option java_outer_classname = "InfraObjectsProto";
+option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/core";
+
+message Infra {
```

Comment:
nit, `Infrastructure`? `DeployedInfrastructure`?

Reply:
Also, add docs for these messages? A couple of lines on how they should be used, interpreted etc?